### ISSUE TITLE
Replace or augment set type with list of set type keywords.

### ISSUE THREAD

**Issue by taw** [mtgjson/mtgjson#412]
Both mtg.wtf and [scryfall](https://scryfall.com/docs/syntax#sets) have overlapping concept of "set types" so a set can have multiple "types", but mtgjson just has one type per set.

Here's a quote from SF docs:

> _You can search for cards based on the type of product they appear in. This includes some high-level categories (st:boxset, st:multiplayer, st:beginner), the primary product types (st:core and st:expansion), series of products (st:masters, st:unset, st:portal, st:commander, st:duel_deck, st:from_the_vault, or st:premium_deck) or categories of promo cards (st:fnm, st:judge_gift, st:arena_league, st:player_rewards, st:media_insert, st:tourney, st:gateway, st:wpn, st:instore, st:convention, st:set_promo, st:duels, st:event, st:premiere_shop, or st:masterpiece — whew!)_

I don't have nice docs for mtg.wtf, but it has some of mtgjson set types, some of same as SF, and few other `st:` like [`st:2hg`](http://mtg.wtf/card?q=st%3A2hg), [st:modern](http://mtg.wtf/card?q=st%3Amodern) etc.

Basically I think Magic has so many weird sets, that it would make sense to have multiple set type tags per set. If that idea sounds sensible, we could come up with a list.

**Comment by ZeldaZach** [comment]
I'm all for it, but I'll need to have a source of truth for such a list

**Comment by taw** [comment]
Well, scryfall has those queries, don't they expose them in their API somehow?

A few of them are a bit questionable, but we can deal with them later.

To be honest I'm not even sure who else could maintain it, since a lot of unofficial promo sets are basically their idea.

**Comment by ZeldaZach** [comment]
If you know of the queries, do let me know. I don't myself

**Comment by taw** [comment]
Hmm, https://scryfall.com/docs/api/sets says it's only one set type, so we'd need to poke them to provide everything their `st:` does.

**Comment by ZeldaZach** [comment]
Hey @csuhta any chance we'll see the full list of set types?

**Comment by csuhta** [comment]
The set `type` on Scryfall indicates why the set exists (for the most part). That will only ever have one value. 

The other search terms like `st:wpn` are actually aliases for [`is:wpn`](https://scryfall.com/search?q=is%3Awpn) which are boolean flags we set on individual card prints. Those flags aren't exposed fully via the API yet due to their messiness/un-final naming scheme. We plan to expose them eventually but I can't provide an estimate on that.
